demos/ag-alert-backend-lastbyte-resp/azuredeploy.json

```diff
@@ -0,0 +1,67 @@
+{
+   "$schema":"https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+   "contentVersion":"1.0.0.0",
+   "parameters":{
+      "alertName":{
+         "type":"String"
+      },
+      "applicationGatewayResourceID":{

```

Comment:
make the same changes here as in #12205

Reply:
https://github.com/Azure/azure-quickstart-templates/pull/12205#discussion_r812779879